src/GitHub.App/Caches/SharedCache.cs

```diff
@@ -104,7 +105,7 @@ static IBlobCache GetBlobCacheWithFallback(Func<IBlobCache> blobCacheFunc, strin
             }
             catch (Exception e)
             {
-                log.Error(string.Format(CultureInfo.InvariantCulture, "Failed to set the {0} cache.", cacheName), e);
+                log.Error(e, "Failed to set the {cacheName} cache.", cacheName);
```

Comment:
Nitpick, but the latest Serilog Analyzer has a lint for non-`PascalCase` property names. (Personally I do like camel case, but Pascal case is definitely the convention.)